### PR TITLE
Cleanup sxsJitStartup from codemanager

### PR DESCRIPTION
**Pull request by gkhanna79**
@jkotas PTAL - we will only look for "jitStartup" and "getJit" exports from both the runtime and crossgen.

FYI - @pgavlin

### CODE REVIEW

**Comment by jkotas** [comment]
LGTM

**Comment by gkhanna79** [comment]
@dotnet-bot test Windows_NT Checked Arm64

File: src/vm/codeman.cpp [hunk 1]
```diff
@@ -1439,6 +1439,8 @@ static void LoadAndInitializeJIT(LPCWSTR pwzJitName, OUT HINSTANCE* phJit, OUT I
 
         EX_TRY
         {
+            bool fContinueToLoadJIT = false;
+#if !defined(FEATURE_CORECLR)
```
Comment (BruceForstall) [inline src/vm/codeman.cpp]: In the JIT, this is determined by SELF_NO_HOST. So I assume the JIT will be built with SELF_NO_HOST defined for all cases of FEATURE_CORECLR defined? But it also looks like sxsJitStartup in the JIT does nothing if SELF_NO_HOST is defined, so why is this necessary?

Comment (gkhanna79) [inline src/vm/codeman.cpp]: This is being done to ensure consistency for CoreCLR based JIT contract.

> So I assume the JIT will be built with SELF_NO_HOST defined for all cases of FEATURE_CORECLR defined

This is already true in the open.
